src/cljam/bam/reader.clj

```diff
@@ -17,6 +18,40 @@
   (close [this]
     (.close ^Closeable (.reader this))))
 
+(declare read-alignments-sequentially*
+         read-alignments*
+         read-blocks-sequentially*
+         read-blocks*)
+
+(extend-type BAMReader
```

Comment:
Thanks to your review.
I fixed this by https://github.com/chrovis/cljam/pull/45/commits/f444f0249fa2e8046b8a48030943157e53f4ddc3 .